src/extrinsic/xt_primitives.rs

```diff
@@ -21,12 +21,16 @@ use sp_std::prelude::*;
 use std::fmt;
 
 use codec::{Compact, Decode, Encode};
-use indices::address::Address;
-use node_primitives::{AccountId, AccountIndex};
-use primitive_types::H256;
+//use indices::address::Address;
+use sp_core::H256;
 use sp_core::blake2_256;
 use sp_runtime::{generic::Era, MultiSignature};
-pub type GenericAddress = Address<AccountId, AccountIndex>;
+
+pub use sp_runtime::AccountId32 as AccountId;
+
+pub type AccountIndex = u64;
+
+pub type GenericAddress = AccountId; //Address<AccountId, AccountIndex>;
```

Comment:
why do that redundant rename? I think it would be better to use AccountId

Reply:
yes, but the problem is that we will want to get back to generic in order to support polkadot/kusama  chains as well which have a different definition.